services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -54,6 +54,7 @@ async def progress_message_parser(app: web.Application, data: bytes) -> None:
                 {
                     "event_type": SOCKET_IO_NODE_UPDATED_EVENT,
                     "data": {
+                        "project_id": project["uuid"],
```

Comment:
not too sure why this change is here, but I guess it does not hurt??

Reply:
this is part of the preparations for the iterator (therefore pre-iterator in the title) 

When running a meta-project, it will run tasks from different iteration projects simultaneously. Therefore a new situation arises in which the active project is not the only one that runs and if the front-end wants to compute the progress of the meta-projects it needs to add up each progress of the iteration projects